### PR TITLE
docs: Update documentation to match changes in UI.

### PR DESCRIPTION
**Pull request by blp**
This updates both text and screenshots.

### CODE REVIEW

File: docs/docs/tour/run-pipeline.gif
Comment (gz) [inline docs/docs/tour/run-pipeline.gif:1]: would be good to record a run where throughput doesnt turn negative  -- sorry about that, a page reload before run should fix it

**Review by gz — APPROVED** [review]

**Comment by mohsinbeg** [comment]
It would be good to use the new logo in the screenshots. It's at https://github.com/feldera/website/tree/main/static/img/logo

File: docs/docs/tour/run-pipeline.gif
Comment (blp) [reply docs/docs/tour/run-pipeline.gif:1]: Oh.

It's painful to reproduce this one but I'll go do it again.

Comment (blp) [reply docs/docs/tour/run-pipeline.gif:1]: OK, done.

**Comment by blp** [comment]
> It would be good to use the new logo in the screenshots. It's at https://github.com/feldera/website/tree/main/static/img/logo

Let's put it in the UI before we try to put it in screenshots of the UI.

**Comment by gz** [comment]
logo is updated in this PR https://github.com/feldera/dbsp/pull/501

**Pull request merged**